### DISCUSSION TITLE
Руководство ПЛИ, тело документа, 1.2. Секция "Информация об исследованных материалах" ("Исследованные материалы") SPECIMENS

### DISCUSSION THREAD

**Discussion by eugenekogan-spb** [fhir-ru/core#281] [Лабораторная РГ]
Секция предназначена для указания сведений о заборе материала и образцах, полученных в процессе обработки материала, с указанием даты и времени забора, доставки и обработки материала, а также исполнителей.
Если медицинский работник присутствует в блоке заголовка "Сведения о документируемом событии" documentationOf, и у него имеется уникальный идентификатор в МИС, то допускается указание только уникального идентификатора исполнителя в качестве ссылки на его подробное описание в блоке documentationOf. Иначе данные об исполнителе указываются полностью.
Идентификаторы, присвоенные материалу и образцу в данной секции, будут использоваться как ссылки, направляющие на их описание в данной секции из кодированных элементов в секции "Результаты лабораторных исследований" ("Результаты проведенных исследований").
В данную секцию включается следующая информация:
• Дата и время поступления материала;
• Дата и время забора материала (при наличии информации);
• Сведения о материале исследования:
o Уникальный идентификатор исследуемого материала в МИС; o Код исследуемого материала;
o Объем исследуемого материала (при наличии информации); o Комментарий к исследуемому материалу (опционально);
o Уточнение места взятия материала; o Анатомическая локализация;
o Латеральность;
• Сведения о медицинском работнике, осуществившем забор материала для исследования (опционально);
• Дата и время обработки материала;
• Сведения об образце исследования:
o Уникальный идентификатор исследуемого образца в МИС; o Код исследуемого образца;
o Объем исследуемого образца (при наличии информации); o Комментарий к исследуемому образцу (опционально);
• Сведения о медицинском работнике, осуществившем обработку материала
для исследования (опционально).

<img width="632" alt="image" src="https://github.com/user-attachments/assets/868a0ea1-7a37-4787-8011-dc134ec35d62">

**Comment by eugenekogan-spb** [comment]
1.2.1. Статус, время поступления материала

У3-1: Секция "Информация об исследованных материалах" ОБЯЗАНА иметь хотя бы один [1..*] элемент entry, содержащий сведения об исследованных материалах, который:
1. ОБЯЗАН иметь один [1..1] элемент organizer (атрибут @classCode должен иметь значение "CLUSTER", атрибут @moodCode должен иметь значение "EVN"), который:
1.ОБЯЗАН иметь один [1..1] элемент statusCode (атрибут @code должен иметь значение "completed");
2.ОБЯЗАН иметь один [1..1] элемент effectiveTime (атрибут @value должен быть представлен с не пустым значением), содержащий дату и время поступления материала;

**Reply by eugenekogan-spb** [reply]
```
* receivedTime = "2022-11-15T07:30:00+04:00"
```
Статус - константа, поэтому может не кодироваться в FHIR.

**Comment by eugenekogan-spb** [comment]
1.2.2. Информация о заборе материала

3. ОБЯЗАН иметь один [1..1] элемент component, содержащий сведения о заборе и обработке материала, который:
1. ОБЯЗАН иметь один [1..1] элемент procedure (атрибут @classCode должен иметь значение "PROC", атрибут @moodCode должен иметь значение "EVN"), который:
1. ДОЛЖЕН иметь один [1..1] элемент code (атрибут @nullFlavor должен иметь значение "NI");
2. ОБЯЗАН иметь один [1..1] элемент statusCode (атрибут @code должен иметь значение "completed");
3. ДОЛЖЕН иметь один [1..1] элемент effectiveTime, содержащий дату и время забора материала (если дата и время забора материала известны, элемент обязан быть представлен). ЕСЛИ забор материала осуществлялся одномоментно, а не в течение некоторого интервала времени, ТО атрибут effectiveTime/@value должен быть представлен с не пустым значением. Данный элемент:
1. МОЖЕТ иметь один [0..1] элемент low (атрибут @value должен быть представлен с не пустым значением), содержащий дату и время начала забора материала. Возможны следующие альтернативы:
1. ЕСЛИ материал собирался некоторый интервал времени, ТО элемент обязан быть представлен;
2. ЕСЛИ иначе, ТО элемент должен отсутствовать;
2. МОЖЕТ иметь один [0..1] элемент high (атрибут @value должен быть представлен с не пустым значением), содержащий дату и время окончания забора материала. Возможны следующие альтернативы:
1. ЕСЛИ материал собирался некоторый интервал времени, ТО элемент обязан быть представлен;
2. ЕСЛИ иначе, ТО элемент должен отсутствовать;

**Reply by eugenekogan-spb** [reply]
Кодируется атрибутами specimen.collectedDateTime или specimen.collectedPeriod. 

```
* collection
  * collectedDateTime = "2022-11-15T06:30:00+04:00"
```

**Comment by eugenekogan-spb** [comment]
1.2.3. Информация о материале исследования

4. ОБЯЗАН иметь один [1..1] элемент specimen, содержащий сведения о материале исследования, который:
1.ОБЯЗАН иметь один [1..1] элемент specimenRole, который:
1.1.ОБЯЗАН иметь один [1..1] элемент id (атрибуты @root, @extension должны быть представлены с не пустыми значениями; атрибут @root должен быть сформирован по правилу формирования идентификаторов материалов и образцов для лабораторных исследований, т.е. "OID_медицинской_организации. 100.НомерМИС.НомерЭкзМИС.66", атрибут @extension должен соответствовать идентификатору материала для лабораторного исследования), содержащий уникальный идентификатор исследуемого материала в МИС;
1.2.ОБЯЗАН иметь один [1..1] элемент specimenPlayingEntity (атрибут @classCode должен иметь значение "ENT", атрибут @determinerCode должен иметь значение "INSTANCE"), который:
1.2.1.ОБЯЗАН иметь один [1..1] элемент code (атрибут @codeSystem должен иметь значение "1.2.643.5.1.13.13.99.2.1019", атрибут @codeSystemName должен иметь значение "Федеральный справочник лабораторных исследований. Справочник лабораторных материалов", атрибуты @code, @codeSystemVersion, @displayName должны быть представлены с не пустыми значениями), содержащий код исследуемого материала, который:
1.2.1.1.МОЖЕТ иметь один [0..1] элемент translation (атрибут @codeSystem должен иметь значение "1.2.643.5.1.13.13.99.2.1064", атрибут @codeSystemName должен иметь значение "Федеральный справочник лабораторных исследований. Уточнение места взятия материала", атрибуты @code, @codeSystemVersion, @displayName должны быть представлены с не пустыми значениями), содержащий уточнение места взятия материала;
1.2.1.2.МОЖЕТ иметь один [0..1] элемент translation (атрибут @codeSystem должен иметь значение "1.2.643.5.1.13.13.11.1477", атрибут @codeSystemName должен иметь значение "Анатомические локализации", атрибуты @code, @codeSystemVersion, @displayName должны быть представлены с не пустыми значениями), содержащий анатомическая локализация, который:
1.2.1.2.1. МОЖЕТ иметь один [0..1] элемент translation (атрибут @codeSystem должен иметь значение "1.2.643.5.1.13.13.99.2.778", атрибут @codeSystemName должен иметь значение "Латеральность", атрибуты @code, @codeSystemVersion, @displayName должны быть представлены с не пустыми значениями), содержащий латеральность;
1.2.1.2.2. ДОЛЖЕН иметь один [1..1] элемент quantity (атрибуты @value, @unit должны быть представлены с не пустыми значениями; атрибут @value должен иметь значение объема исследуемого материала, атрибут @unit должен иметь значение по справочнику "Единицы измерения" (OID: 1.2.643.5.1.13.13.11.1358), поле "Код UCUM"), содержащий объем исследуемого материала, который:
1.2.1.2.2.1.ОБЯЗАН иметь один [1..1] элемент translation (атрибут @codeSystem должен иметь значение "1.2.643.5.1.13.13.11.1358", атрибут @codeSystemName должен иметь значение "Единицы измерения", атрибуты @value, @code, @codeSystemVersion, @displayName должны быть представлены с не пустыми значениями; атрибут @value должен иметь значение объема исследуемого материала);
1.2.1.2.3.МОЖЕТ иметь один [0..1] элемент desc, содержащий комментарий к исследуемому материалу, который должен иметь не пустое наполнение;

**Reply by eugenekogan-spb** [reply]
```
* identifier
  * system                   = "urn:oid:1.2.643.5.1.13.13.12.2.22.27143.100.1.1.66"
  * value                    = "1234567890"

* type                       = urn:oid:1.2.643.5.1.13.13.99.2.1019#2   "Кровь венозная"
```
анатомическая локализация, латеральность, объем материала в примере СЭМД отсутствуют. См ниже примеры, придуманные В.Родионовым.

**Comment by eugenekogan-spb** [comment]
1.2.4. Информация о медицинских работниках, осуществивших забор материала

5. МОЖЕТ иметь произвольное количество [0..*] элементов performer, содержащих сведения о медицинском работнике, осуществившем забор материала для исследования, который:
1. ОБЯЗАН иметь один [1..1] элемент assignedEntity (если медицинский работник присутствует в блоке заголовка "Сведения о документируемом событии" documentationOf, и у него имеется уникальный идентификатор в МИС, то допускается указание только уникального идентификатора медицинского работника в элементе assignedEntity\id в качестве ссылки на его подробное описание в блоке documentationOf)...

**Reply by eugenekogan-spb** [reply]
```
* collection
  * collector
    * identifier
      * system = "1.2.643.5.1.13.13.12.2.22.1576.100.1.1.70"
      * value = "34678"
      * type
        * text = "Уникальный идентификатор медицинского работника в МИС"
```

**Comment by eugenekogan-spb** [comment]
1.2.5. Образцы, полученные в ходе обработки материала

6. МОЖЕТ иметь произвольное количество элементов entryRelationship (атрибут @typeCode должен иметь значение "REFR"), содержащих описание образца, полученного в ходе обработки материала, который:
6.1. ОБЯЗАН иметь один [1..1] элемент procedure (атрибут @classCode должен иметь значение "PROC", атрибут @moodCode должен иметь значение "EVN"), который:

6.1.1. ДОЛЖЕН иметь один [1..1] элемент code (атрибут @nullFlavor должен иметь значение "NI");

6.1.2. ОБЯЗАН иметь один [1..1] элемент statusCode (атрибут @code должен иметь значение "completed");

6.1.3. ОБЯЗАН иметь один [1..1] элемент effectiveTime, содержащий дату и время обработки материала. ЕСЛИ обработка материала осуществлялась одномоментно, а не в течение некоторого интервала времени, ТО атрибут effectiveTime/@value должен быть представлен с не пустым значением. Данный элемент:
6.1.3.1. МОЖЕТ иметь один [0..1] элемент low (атрибут @value должен быть представлен с не пустым значением), содержащий дату и время начала обработки материала. Возможны следующие альтернативы:
- ЕСЛИ материал обрабатывался некоторый интервал времени, ТО элемент обязан быть представлен;
- ЕСЛИ иначе, ТО элемент должен отсутствовать;
6.1.3.2. МОЖЕТ иметь один [0..1] элемент high (атрибут @value должен быть представлен с не пустым значением), содержащий дату и время окончания обработки материала. Возможны следующие альтернативы:
- ЕСЛИ материал обрабатывался некоторый интервал времени, ТО элемент обязан быть представлен;
- ЕСЛИ иначе, ТО элемент должен отсутствовать;

6.1.4.ОБЯЗАН иметь один [1..1] элемент specimen, содержащий
сведения об образце исследования, который:
6.1.4.1. ОБЯЗАН иметь один [1..1] элемент specimenRole, который:
6.1.4.1.1. ОБЯЗАН иметь один [1..1] элемент id (атрибуты @root, @extension должны быть представлены с не пустыми значениями; атрибут @root должен быть сформирован по правилу формирования идентификаторов материалов и образцов для лабораторных исследований, т.е. "OID_ медицинской_организации.100.НомерМИС.НомерЭкзМИС.66", атрибут @extension должен соответствовать идентификатору образца для лабораторного исследования), содержащий уникальный идентификатор исследуемого образца в МИС;
6.1.4.1.2. ОБЯЗАН иметь один [1..1] элемент specimenPlayingEntity (атрибут @classCode должен иметь значение "ENT", атрибут @determinerCode должен иметь значение "INSTANCE"), который:
6.1.4.1.2.1.ОБЯЗАН иметь один [1..1] элемент code (атрибут @codeSystem должен иметь значение "1.2.643.5.1.13.13.11.1081", атрибут @codeSystemName должен иметь значение "Федеральный справочник лабораторных исследований. Справочник лабораторных материалов и образцов", атрибуты @code, @codeSystemVersion, @displayName должны быть представлены с не пустыми значениями), содержащий код исследуемого образца;
6.1.4.1.2.2. ДОЛЖЕН иметь один [1..1] элемент quantity (атрибуты @value, @unit должны быть представлены с не пустыми значениями; атрибут @value должен иметь значение объем исследуемого образца, атрибут @unit должен иметь значение по справочнику "Единицы измерения" (OID: 1.2.643.5.1.13.13.11.1358), поле "Код UCUM"), содержащий объем исследуемого образца, который:
6.1.4.1.2.2.1. ОБЯЗАН иметь один [1..1] элемент translation (атрибут @codeSystem должен иметь значение "1.2.643.5.1.13.13.11.1358", атрибут @codeSystemName должен иметь значение "Единицы измерения", атрибуты @value, @code, @codeSystemVersion, @displayName должны быть представлены с не пустыми значениями; атрибут @value должен иметь значение объем исследуемого образца);
6.1.4.1.2.3. МОЖЕТ иметь один [0..1] элемент desc, содержащий комментарий к исследуемому образцу, который должен иметь не пустое наполнение;

6.1.5. МОЖЕТ иметь произвольное количество [0..*] элементов performer, содержащих сведения о медицинском работнике, осуществившем обработку материала для исследования...

**Reply by eugenekogan-spb** [reply]
В примере СЭМД образец объединен с материалом. Его единственная характеристика - сведения о контейнере.
```
* note
  * text = "Пластиковая пробирка для гематологических исследований с K3-ЭДТА 
(шрихкод 1234567890)"
```

**Comment by Vrodvi** [comment]
Пытаемся сформировать переложение наполнения секции "Информация об исследованных материалах" в формат FHIR-ресурсов.

Для начала составим логическую модель сведений, содержащихся в данной секции документа:
```
В данную секцию включается следующая информация:
• Дата и время поступления материала;
• Дата и время забора материала (при наличии информации);
• Сведения о материале исследования:
	o Уникальный идентификатор исследуемого материала в МИС;
	o Код исследуемого материала;
	o Объем исследуемого материала (при наличии информации);
	o Комментарий к исследуемому материалу (опционально);
	o Уточнение места взятия материала;
	o Анатомическая локализация;
	o Латеральность;
• Сведения о медицинском работнике, осуществившем забор материала для
исследования (опционально);
• Дата и время обработки материала;
• Сведения об образце исследования:
	o Уникальный идентификатор исследуемого образца в МИС;
	o Код исследуемого образца;
	o Объем исследуемого образца (при наличии информации);
	o Комментарий к исследуемому образцу (опционально);
• Сведения о медицинском работнике, осуществившем обработку материала для исследования (опционально).
```

Логическую модель составим через Logical-ресурсы FHIR (https://hl7.org/fhir/logical.html):
```
// Само логическое описание моделей данных

Logical: Specimen
* receivedTime    1..1  dateTime                     "Дата и время поступления материала в лабораторию"
* collected[x]    1..1  dateTime or Period           "Дата и время [или интервал] времени забора материала"
* identifier      1..1  Identifier                   "Уникальный идентификатор исследуемого материала в МИС"
* specimenType    1..1  CodeableConcept              "Код исследуемого материала"
* collectionSite  0..1  CodeableConcept              "Уточнение места взятия материала"
* bodySite        0..1  CodeableConcept              "Анатомическая локализация"
  * laterality    0..1  CodeableConcept              "Латеральность"
* quantity        1..1  SimpleQuantity               "Объем исследуемого материала"
* note            0..1  string                       "Комментарий к исследуемому материалу"
* collector       0..*  Reference(PractitionerRole)  "Сведения о медицинском работнике, осуществившем забор материала для исследования"
* samples         0..*  Reference(Sample)            "Описание образца, полученного в ходе обработки материала"

Logical: Sample
Characteristics: #can-be-target
* processed[x]    1..1  dateTime or Period           "Дата и время [или интервал] обработки материала [для получения образца]"
* identifier      1..1  Identifier                   "Уникальный идентификатор исследуемого образца в МИС"
* sampleType      1..1  CodeableConcept              "Код исследуемого образца"
* quantity        1..1  SimpleQuantity               "Объем исследуемого образца"
* note            0..1  string                       "Комментарий к исследуемому образцу"
* processor       0..*  Reference(PractitionerRole)  "Сведения о медицинском работнике, осуществившем обработку материала [для получения образца] для исследования"

// Примеры экземпляров для обозначеных логических моделей данных

Instance: SpecimenUrine
InstanceOf: Specimen
Usage: #example
* receivedTime      = "2023-08-01T11:00:00.0Z"
* collectedDateTime = "2023-08-01T11:11:00.0Z"
* identifier
  * system          = "urn:oid:1.2.643.5.1.13.13.12.2.22.27143.100.1.1.66"
  * value           = "0483610201"
* specimenType      = urn:oid:1.2.643.5.1.13.13.99.2.1019#9   "Моча"
* collectionSite    = urn:oid:1.2.643.5.1.13.13.99.2.1064#16  "Катетер урологический"
* bodySite          = urn:oid:1.2.643.5.1.13.13.11.1477#52    "Мочеточник"
  * laterality      = urn:oid:1.2.643.5.1.13.13.99.2.778#2    "Справа"
* quantity          = 50 urn:oid:1.2.643.5.1.13.13.11.1358#16 "мл"
* note              = "Пластиковый контейнер (шрихкод 0483610201)"
* collector         = Reference(ExamplePractitionerRole)
* samples[0]        = Reference(SampleCulture1)
* samples[+]        = Reference(SampleCulture2)

Instance: SampleCulture1
InstanceOf: Sample
Usage: #example
* processedPeriod
  * start         = "2023-08-01T11:11:00.0Z"
  * end           = "2023-08-01T11:22:00.0Z"
* identifier
  * system          = "urn:oid:1.2.643.5.1.13.13.12.2.22.27143.100.1.1.66"
  * value           = "0483610201001"
* sampleType        = urn:oid:1.2.643.5.1.13.13.11.1081#115    "Культура микроорганизмов"
* quantity          = 10000 urn:oid:1.2.643.5.1.13.13.11.1358#519 "КОЕ/мл"
* note              = "Культура микроорганизмов (10000 КОЕ/мл) (штрихкод 0483610201001)"
* processor         = Reference(ExamplePractitionerRole)

Instance: SampleCulture2
InstanceOf: Sample
Usage: #example
* processedPeriod
  * start         = "2023-08-01T11:11:00.0Z"
  * end           = "2023-08-01T11:22:00.0Z"
* identifier
  * system          = "urn:oid:1.2.643.5.1.13.13.12.2.22.27143.100.1.1.66"
  * value           = "0483610201002"
* sampleType        = urn:oid:1.2.643.5.1.13.13.11.1081#115    "Культура микроорганизмов"
* quantity          = 10000 urn:oid:1.2.643.5.1.13.13.11.1358#519 "КОЕ/мл"
* note              = "Культура микроорганизмов (10000 КОЕ/мл) (штрихкод 0483610201002)"
* processor         = Reference(ExamplePractitionerRole)

Instance: ExamplePractitionerRole
InstanceOf: PractitionerRole
Usage: #example
```

Составим графическое представление этих логических моделей в виде ER-диаграммы (используем https://plantuml.com/ie-diagram):
```
@startuml
' hide the spot
' hide circle

' avoid problems with angled crows feet
skinparam linetype ortho

entity "Specimen" as sp {
  receivedTime \t: 1..1 dateTime
  collected[x] \t: 1..1 dateTime or Period
  *identifier \t\t: 1..1 Identifier
  specimenType \t: 1..1 CodeableConcept
  collectionSite \t: 0..1 CodeableConcept
  bodySite \t\t: 0..1 CodeableConcept
  laterality \t\t: 0..1 CodeableConcept
  quantity \t\t: 1..1 SimpleQuantity
  note \t\t: 0..1 string
  collector \t\t: 0..* Reference{PractitionerRole}
  samples \t\t: 0..* Reference{Sample}
} 

entity "Sample" as sa {
  processed[x] \t: 1..1dateTime or Period
  *identifier \t\t: 1..1 Identifier
  sampleType \t: 1..1 CodeableConcept
  quantity \t\t: 1..1 SimpleQuantity
  note \t\t: 0..1 string
  processor \t: 0..* Reference{PractitionerRole}
}
 
sp ||..o{ sa
@enduml
```
Результирующая визуализация:
![МатериалыОбразцы](https://github.com/user-attachments/assets/dea6dc30-bbd5-4b4f-a6fa-b28274021839)


Теперь можно разложить элементы логических моделей по подходящим FHIR ресурсам (в нашем случае и материал и образец оформляются через ресурс Specimen):
```
Instance: SpecimenUrine
InstanceOf: Specimen
Usage: #example
* identifier
  * system                   = "urn:oid:1.2.643.5.1.13.13.12.2.22.27143.100.1.1.66"
  * value                    = "0483610201"
* type                       = urn:oid:1.2.643.5.1.13.13.99.2.1019#9   "Моча"
* subject                    = Reference(ExamplePatient) "Иванов Иван"
* receivedTime               = "2023-08-01T11:00:00.0Z"
* collection
  * collector                = Reference(ExamplePractitionerRole)
  * collectedDateTime        = "2023-08-01T11:11:00.0Z"
  * quantity                 = 50 urn:oid:1.2.643.5.1.13.13.11.1358#16 "мл"
  * bodySite 
    * concept                = urn:oid:1.2.643.5.1.13.13.11.1477#52    "Мочеточник"
    * extension[0]
      * url                  = "laterality"
      * valueCodeableConcept = urn:oid:1.2.643.5.1.13.13.99.2.778#2    "Справа"
  * extension[0]
    * url                    = "collectionSite"
    * valueCodeableConcept   = urn:oid:1.2.643.5.1.13.13.99.2.1064#16  "Катетер урологический"
* note
  * text                     = "Пластиковый контейнер (шрихкод 0483610201)"

Instance: SampleCulture1
InstanceOf: Specimen
Usage: #example
* parent            = Reference(SpecimenUrine)
* identifier
  * system          = "urn:oid:1.2.643.5.1.13.13.12.2.22.27143.100.1.1.66"
  * value           = "0483610201001"
* type              = urn:oid:1.2.643.5.1.13.13.11.1081#115    "Культура микроорганизмов"
* subject           = Reference(ExamplePatient) "Иванов Иван"
* collection
  * collector       = Reference(ExamplePractitionerRole)
  * collectedPeriod 
    * start         = "2023-08-01T11:11:00.0Z"
    * end           = "2023-08-01T11:22:00.0Z"
  * quantity        = 10000 urn:oid:1.2.643.5.1.13.13.11.1358#519 "КОЕ/мл"
* note
  * text            = "Культура микроорганизмов (10000 КОЕ/мл) (штрихкод 0483610201001)"

Instance: SampleCulture2
InstanceOf: Specimen
Usage: #example
* parent            = Reference(SpecimenUrine)
* identifier
  * system          = "urn:oid:1.2.643.5.1.13.13.12.2.22.27143.100.1.1.66"
  * value           = "0483610201002"
* type              = urn:oid:1.2.643.5.1.13.13.11.1081#115    "Культура микроорганизмов"
* subject           = Reference(ExamplePatient) "Иванов Иван"
* collection
  * collector       = Reference(ExamplePractitionerRole)
  * collectedPeriod 
    * start         = "2023-08-01T11:11:00.0Z"
    * end           = "2023-08-01T11:22:00.0Z"
  * quantity        = 10000 urn:oid:1.2.643.5.1.13.13.11.1358#519 "КОЕ/мл"
* note
  * text            = "Культура микроорганизмов (10000 КОЕ/мл) (штрихкод 0483610201002)"

Instance: ExamplePractitionerRole
InstanceOf: PractitionerRole
Usage: #example

Instance: ExamplePatient
InstanceOf: Patient
Usage: #example
```
(видим, что часть данных, таких как "Уточнение места взятия материала" и "Латеральность", пришлось разместить в локальных расширениях, т.к. для них отсутствуют стандартные атрибуты в FHIR-ресурсах)

**Comment by eugenekogan-spb** [comment]
Пример секции в спецификации СЭМД ПЛИ

<img width="761" alt="image" src="https://github.com/user-attachments/assets/6ced6e72-46ed-4d3b-a3a7-da05900b25e6">
<img width="760" alt="image" src="https://github.com/user-attachments/assets/69c85c75-5f8e-4ddc-a35b-0758472cb549">

**Reply by eugenekogan-spb** [reply]
В примере, приведенном в СЭМД, материал и образец представлены в виде единой сущности и выложены в разделе "материал". 

```
Instance: Specimen_Blood
Id: specimen-blood
InstanceOf: Specimen
Usage: #example
* identifier
  * system                   = "urn:oid:1.2.643.5.1.13.13.12.2.22.27143.100.1.1.66"
  * value                    = "1234567890"

* type                       = urn:oid:1.2.643.5.1.13.13.99.2.1019#2   "Кровь венозная"

* subject                    = Reference(patient-maria-antuanetta) "Иванов Иван"

* receivedTime               = "2022-11-15T07:30:00+04:00"

* collection
  * collector
    * identifier
      * system = "1.2.643.5.1.13.13.12.2.22.1576.100.1.1.70"
      * value = "34678"
      * type
        * text = "Уникальный идентификатор медицинского работника в МИС"

* note
  * text                     = "Пластиковая пробирка для гематологических исследований с K3-ЭДТА
(шрихкод 1234567890)"
```

Медицинский работник при этом представлен только идентификатором.
